curriculum/challenges/english/25-front-end-development/lab-product-landing-page/587d78af367417b2b2512b04.md

```diff
@@ -260,7 +277,7 @@ const usesFlex = cssRules.find(rule => hasFlex(rule))
 const usesFlexMedia = mediaRules.find(mediaRule => {
   return [...mediaRule.cssRules].find(rule => hasFlex(rule))
 })
-assert(usesFlex || usesFlexMedia)
+assert.isTrue(usesFlex || usesFlexMedia);
```

Comment:
```suggestion
assert.exists(usesFlex || usesFlexMedia);
```

With `isTrue` it's failing as `find` returns the found element or `undefined`, so here something like `isDefined` or `exists` would work better than `isTrue`